Added Korean and Mandarin Chinese

imogen-mcara opened the pull request:
As with my first pull request, these are the datasets and metadata for kin terms in Mandarin and Korean sourced to the best of my ability.

imogen-mcara committed:
Added Korean and Mandarin Chinese

As with my first pull request, these are the datasets and metadata for kin terms in Mandarin and Korean sourced to the best of my ability.

SamPassmore commented:
Thanks for adding these, Imogen. Everything worked well, I only had to change one thing. 

In your raw data files, you had two rows that just said column1, column2, column3, etc (see picture, or look at the files changed tab above). I would guess these are added by the software you are using, but its worth opening the files in a text editor (like notepad) to see if anything has been added automatically before uploading. I have removed them now, so everything is done! These will move into the website once we update the main database - but I will do that in a few months after I have added some more languages of my own. 

![Screenshot 2024-08-15 at 10 06 23 am](https://github.com/user-attachments/assets/b71d7095-d1a4-423d-90c9-32071555b104)

imogen-mcara commented:
Hi Sam,

Thank you for sorting that out and for the tip, I’ll keep that in mind!
________________________________
From: SamPassmore ***@***.***>
Sent: Thursday, August 15, 2024 1:13 AM
To: kinbank/varikin ***@***.***>
Cc: Imogen McAra ***@***.***>; Author ***@***.***>
Subject: Re: [kinbank/varikin] Added Korean and Mandarin Chinese (PR #1)


Thanks for adding these, Imogen. Everything worked well, I only had to change one thing.

In your raw data files, you had two rows that just said column1, column2, column3, etc (see picture, or look at the files changed tab above). I would guess these are added by the software you are using, but its worth opening the files in a text editor (like notepad) to see if anything has been added automatically before uploading. I have removed them now, so everything is done! These will move into the website once we update the main database - but I will do that in a few months after I have added some more languages of my own.

Screenshot.2024-08-15.at.10.06.23.am.png (view on web)<https://github.com/user-attachments/assets/b71d7095-d1a4-423d-90c9-32071555b104>

—
Reply to this email directly, view it on GitHub<https://github.com/kinbank/varikin/pull/1#issuecomment-2290108987>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BJUH2HNUEGPLQTROSGESEDDZRPXD3AVCNFSM6AAAAABL3TRLJCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOJQGEYDQOJYG4>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
